plugins/reporters/web-app-template/src/components/PackagePaths.jsx

```diff
@@ -98,7 +98,7 @@ const PackagePaths = ({ paths }) => {
                         }
                     );
 
-                    stepItems.push({ title: webAppPath.packageName });
+                    stepItems.push({ title: webAppPath.packageId });
```

Comment:
Commit message: "..., which is an ORT Identifier [1] containing type, namespace, name and version". (I'm adding a comma and the version here; but actually, I believe that naming all identifier properties could be omitted.)